keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExceptionHandler.kt

```diff
@@ -86,9 +87,9 @@ class ExceptionHandler(
     return ApiError(CONFLICT, e)
   }
 
-  @ExceptionHandler(ValidationException::class)
+  @ExceptionHandler(ValidationException::class, InvalidVetoException::class)
   @ResponseStatus(BAD_REQUEST)
-  fun onInvalidDeliveryConfig(e: ValidationException): ApiError {
+  fun onInvalidDeliveryConfig(e: Exception): ApiError {
```

Comment:
why was this changed?

Reply:
So that we could do the same thing for multiple exception classes (previously this was only handling the `ValidationException` as a `BAD_REQUEST`